modules/data.atmosphere/R/download.PalEON_ENS.R

```diff
@@ -0,0 +1,62 @@
+##' Download PalEON files
+##'
+##' @name download.PalEON
+##' @title download.PalEON
+##' @export
+##' @param outfolder
+##' @param start_year
+##' @param end_year
+##' 
+##' @author Betsy Cowdery, Mike Dietze
+download.PalEON_ENS <- function(sitename, outfolder, start_date, end_date, overwrite = FALSE, ...) {
+  
+  library(PEcAn.utils)
```

Comment:
This is bad practice. If you _need_ to load the entire package namespace, the way to do that is to add `@import PEcAn.utils` as a roxygen tag. Preferably, if you only need certain functions from that package, the recommended way to do that is to use `@importFrom PEcAn.utils logger.info logger.warn logger.severe`. I still think the best thing to do for code readability is to explicitly do `PEcAn.utils::logger.warn`, etc. at every instance.